Assets/HoloToolkit/Utilities/Editor/ConfigureMenu.cs

```diff
@@ -0,0 +1,187 @@
+
+                string title = "Updates require text serialization of assets";
+                string message = "Unity doesn't provide apis for updating the default quality or enabling VR support.\n\n" +
+                    "Is it ok if we force text serialization of assets so that we can modify the properties directly?";
+
+                bool forceText = EditorUtility.DisplayDialog(title, message, "Yes", "No");
+                if (!forceText)
+                    return;
+
+                EditorSettings.serializationMode = SerializationMode.ForceText;
+            }
+
+            SetFastestDefaultQuality();
+            EnableVirtualReality();
+
+            // Since we went behind Unity's back to tweak some settings we 
+            // need to reload the project to have them take effect
+            bool canReload = EditorUtility.DisplayDialog(
+                "Project reload required!",
+                "Some changes require a project reload to take effect.\n\nReload now?",
+                "Yes", "No");
+
+            if (canReload)
+                EditorApplication.OpenProject(Application.dataPath + "/..");
+        }
+
+        /// <summary>
+        /// Modifies the WSA default quality setting to the fastest
+        /// </summary>
+        private static void SetFastestDefaultQuality()
+        {
+            try
+            {
+                // Find the WSA element under the platform quality list and replace it's value with 0
+                string settingsPath = "ProjectSettings/QualitySettings.asset";
+                string matchPattern = @"(m_PerPlatformDefaultQuality.*Windows Store Apps:) (\d+)";
+                string replacePattern = @"$1 0";
+
+                string settings = File.ReadAllText(settingsPath);
+                settings = Regex.Replace(settings, matchPattern, replacePattern, RegexOptions.Singleline);
+
+                File.WriteAllText(settingsPath, settings);
+            }
+            catch (Exception e)
+            {
+                Debug.LogException(e);
+            }
+        }
+
+        /// <summary>
+        /// Enables virtual reality for WSA and ensures HoloLens is in the supported SDKs.
+        /// </summary>
+        private static void EnableVirtualReality()
+        {
+            try
+            {
+                // Grab the text from the project settings asset file
+                string settingsPath = "ProjectSettings/ProjectSettings.asset";
+                string settings = File.ReadAllText(settingsPath);
+
+                // Set VR for WSA to 1
+                string matchPattern = @"(Metro::VR::enable:) (\d+)";
+                string replacePattern = @"$1 1";
+                settings = Regex.Replace(settings, matchPattern, replacePattern, RegexOptions.Singleline);
+
+                // This is a bit more complex, we're looking for the WSA list of enabled VR devices, then
+                // ensuring that the HoloLens is in that list
```

Comment:
In the project settings file it is stored as "HoloLens". That will display in the UI as "Windows Holographic".